quantus/metrics/complexity/complexity.py

```diff
@@ -264,3 +250,29 @@ def evaluate_instance(
 
         a = np.array(np.reshape(a, newshape), dtype=np.float64) / np.sum(np.abs(a))
         return scipy.stats.entropy(pk=a)
+
+    def evaluate_batch(
+        self, *, x_batch: np.ndarray, a_batch: np.ndarray, **_
```

Comment:
are all these asterisks actually needed? please describe!

Reply:
`*` means the function takes arguments as keywords only, which prevents errors cause by wrong (positional) argument order.